### DISCUSSION TITLE
Suggestions for New Features

### DISCUSSION THREAD

**Discussion by dhairyaostwal** [dhairyaostwal/finCLI#2] [Ideas]
Hey there! Glad you checked out this project. Help suggesting a few features for building a better **finCLI**.

## Already Implemented:

- [x] Get News on BTC, Ethereum, Cryptocurrency, Global Finance
- [x] Indian News output performing well
- [x] Published v1.0 
- [ ] README to be updated
- [ ] SENSEX/NIFTY/Stock price on demand

**Comment by ayush5588** [comment]
How about a feature for information on trending cryptocurrencies along with the prices?

**Reply by dhairyaostwal** [reply]
Sounds nice. Would you mind implementing it? Possibly you'd have to figure out a new API for that. @ayush5588 if you're able to do it before sending a PR, post a small video of its implementation. 

Cheers 🎉

**Reply by ayush5588** [reply]
Yeah sure. If I find an API for that I will implement it and share it's video.

**Reply by dhairyaostwal** [reply]
@ayush5588 Hey, I've also setup a Kanban board for Project, request you to kindly check that out 😄 

https://github.com/dhairyaostwal/finCLI/projects/1

**Reply by ayush5588** [reply]
@dhairyaostwal Looks pretty nice.

**Reply by dhairyaostwal** [reply]
@ayush5588 Hey! How're you doing? Let's build this more and not let it be just a project.
Created an org. page - https://github.com/finCLI

![finCLI-1](https://user-images.githubusercontent.com/50984984/112564768-dbe10200-8e01-11eb-9200-eb6a5ac665ce.png)
